PfMultiplayerCmdlets/NewPFMultiplayerBuild.cs

```diff
@@ -12,6 +12,9 @@ public class NewPFMultiplayerBuild : PFBaseCmdlet
         [Parameter(Mandatory = true)]
         public string BuildName { get; set; }
 
+        [Parameter]
+        public ContainerFlavor ContainerFlavor { get; set; } = ContainerFlavor.ManagedWindowsServerCore;
```

Comment:
ContainerFlavour.ManagedWindowsServerCore = 0 so we could remove the explicit default if you like?